.github/workflows/reusable-web-e2e.yml

```diff
@@ -117,35 +120,64 @@ jobs:
         with:
           working-directory: apps/web
           browser: "${{ env.BROWSER_PATH }}"
-          record: false
+          record: true
+          parallel: true
           install: false
-          parallel: false
           config-file: cypress.config.ts
-          spec: "cypress/tests/**/*.spec-ee.{js,jsx,ts,tsx}"
+          spec: |
+            cypress/tests/**/*.spec.ts
+            ${{ steps.setup.outputs.has_token == 'true' && inputs.ee && 'cypress/tests/**/*.spec-ee.ts' }}
 
-      - name: Cypress run e2e
-        uses: cypress-io/github-action@v6
-        env:
-          NODE_ENV: test
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          CYPRESS_RECORD_KEY: ${{ secrets.CYPRESS_WEB_KEY }}
-          CYPRESS_GITHUB_USER_EMAIL: ${{ secrets.CYPRESS_GITHUB_USER_EMAIL }}
-          CYPRESS_GITHUB_USER_PASSWORD: ${{ secrets.CYPRESS_GITHUB_USER_PASSWORD }}
-          CYPRESS_IS_CI: true
+      - name: Playwright Install
+        working-directory: apps/web
+        run: pnpm playwright:install
+
+      - name: Run Playwright tests
+        working-directory: apps/web
+        run: pnpm playwright:test --shard=${{ matrix.containers }}/${{ matrix.total }}
+
+      - uses: actions/upload-artifact@v4
+        if: ${{ !cancelled() }}
         with:
-          working-directory: apps/web
-          browser: "${{ env.BROWSER_PATH }}"
-          record: false
-          parallel: false
-          install: false
-          config-file: cypress.config.ts
-          spec: "cypress/tests/**/*.spec.{js,jsx,ts,tsx}"
+          name: blob-report-${{ matrix.containers }}
+          path: apps/web/blob-report
+          retention-days: 1
```

Comment:
upload the results of the tests, each shard creates the "blob" of the report, in the next step we merge these blobs to one report